docs/dev_install.rst

```diff
@@ -108,9 +118,31 @@ If you don't have CUDA, go to `this website <https://developer.nvidia.com/cuda-d
 - On **Arch Linux**, you can easily install CUDA via ``pacman -S cuda`` without downloading the installer manually.
 
 
+.. _dev_env_settings:
+
 Setting up Taichi for development
 ---------------------------------
 
+- Set up environment variables for Taichi:
+
+  * On Linux / OS X, please add the following script to your ``~/.bashrc``:
```

Comment:
nit:
```suggestion
  * On Linux / OS X, please add the following script to your rc file (``~/.bashrc``, ``~/.zshrc`` or etc. , same for other occurrences in this documentation):
```
Since we don't want to make assumptions on which shell the users are using (I'm using zsh myself)

Reply:
Me too! It's also heard that OS X users are using `zsh` by default.